src/tests/ftest/vmd/led.yaml

```diff
@@ -0,0 +1,51 @@
+hosts:
+  test_servers: 2
+  test_clients: 1
+timeout: 2700
```

Comment:
Have you considered separate timeouts for each test method?

Reply:
Yes, Reducing the timeout to 300. I don't think it takes more time for the tests to complete.